packages/xo-web/src/xo-app/backup/overview/tab-jobs.js

```diff
@@ -349,6 +350,9 @@ class JobsTable extends React.Component {
               {compression !== undefined && (
                 <Li>{_.keyValue(_('compression'), compression === 'native' ? 'GZIP' : compression)}</Li>
               )}
+              {nRetriesVmBackupFailures !== undefined && nRetriesVmBackupFailures !== 0 && (
```

Comment:
```suggestion
              {nRetriesVmBackupFailures > 0 && (
```
since `undefined >0 === false `
or you may initialiaze nRetriesVmBackupFailures to zero if not defined